Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -904,10 +904,6 @@ private GraphicInputEventData HandlePointerUp(IMixedRealityPointer pointer)
 
             GraphicInputEventData graphicEventData;
             FocusProvider.TryGetSpecificPointerGraphicEventData(pointer, out graphicEventData);
-            
-            GameObject focusedObject = FocusProvider.GetFocusedObject(pointer);
-            graphicEventData.selectedObject = focusedObject;
```

Comment:
graphicEventData had the selected object already in it's .pointerCurrentRaycast.gameObject